content/en/service_management/app_builder/queries.md

```diff
@@ -111,6 +111,22 @@ To provide mocked outputs manually, perform the following steps:
 {{% /collapse-content %}} 
 
 
+## Order of operations
+
+When executing a query, App Builder performs the following steps in the order listed:
+
+1. Checks if there is a **Condition** expression for the query, and if so, checks that the condition is met. If it is not, execution stops.
+2. Evaluates any expressions in **Inputs** to determine the input data for the query.
+3. If the **Debounce** property is set, delays execution for the interval defined by the debounce value. If query inputs or their dependencies update during this time, the current query execution is stopped, and a new one starts from the beginning using the updated inputs.
+   **Note**: If more than one query request occurs within the debounce interval, all requests except the last execution request are canceled.  
```

Comment:
```suggestion
</br>
**Note**: If more than one query request occurs within the debounce interval, all requests except the last execution request are canceled.  
```
I think this should be on the line below? Right now it's not showing up on a new line so this should help, let me know though!